cmd/notifier/config/config.toml

```diff
@@ -13,6 +13,26 @@
         Prefix = "erd"
         Length = 32
 
+[WebSocketConnector]
+    # URL for the WebSocket client/server connection
+    # This value represents the IP address and port number that the WebSocket client or server will use to establish a connection.
+    url = "localhost:22111"
+
+    # This flag describes the mode to start the WebSocket connector. Can be "client" or "server"
+    mode = "server"
```

Comment:
why not client?

Reply:
also, keep consistency for the TOML properties. If the old ones were following the uppercase + camel case pattern, do the same for the new ones (so, for example, `data-marshalller-type` -> `DataMarshallerType`)

Reply:
we use notifier as server, and observer nodes as clients; the same pattern was before with http requests